src/pages/BaseItems.page.js

```diff
@@ -0,0 +1,42 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class BaseItemsPage extends BaseSwagLabPage {
+
+    itemSelector = '.cart_item';
```

Comment:
Its better to stick to data-test attributes everywhere you can. Classes could change.